kiwi/iso_tools/xorriso.py

```diff
@@ -147,16 +147,14 @@ def add_efi_loader_parameters(self) -> None:
         file refer to _create_embedded_fat_efi_image() from
         bootloader/config/grub2.py
         """
-        loader_file = os.sep.join([self.source_dir, self.boot_path, 'efi'])
-        if os.path.exists(loader_file):
```

Comment:
Any reason why the check for the existence of the given loader_file had to be deleted ?

Reply:
If `add_efi_loader_parameters` is called, the file has to exist. The check would only hide bugs.

Reply:
ok makes sense